source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost Mobile technologies. It is a compliment to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+***********************
+Technical  Requirements
+***********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to bon at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+`Network Requirements <https://>`_
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via the public App Stores (i.e. Google Play And Apple App Store), or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the public app stores greatly reduces the deployment time and is our recommended approach. Key benefits include… 
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
+===========
+
+If you desire to customize the applications, or do not want your users downloading the application from the public app stores, you will need to build the apps yourself. 
+
+As the apps are an open source project, it will require a fork, and your team will be responsible for maintaining the fork, as well as keeping the fork up to date with any changes made by Mattermost.
```

Comment:
They do not really need a fork, special care with this. I can just get the source code and build it without the need to fork the repo.

Reply:
Agree with the distinction. This came out in several notes, but I don’t see any documentation that aligns with that particular usage.

Reply:
"To customize the mobile apps, you can fork the open source repository and compile your own versions of the Mattermost mobile applications and Mattermost push proxy server."  https://docs.mattermost.com/mobile/mobile-overview.html#build-the-apps-yourself

Reply:
My knowledge is lacking here. For my own education, what scenario is addressed by building the apps without any changes? When would a customer want to do that?

Reply:
When they  want to deploy their own MPNS for compliance reasons

Reply:
Right. I think we address that in other areas. For this section the focus is guiding a path that is related to a customer version of the mobile apps. Given the audience, and what is being conveyed, this should stay as is.

Reply:
solved in meeting